.github/workflows/openliberty-ci.yml

```diff
@@ -82,27 +81,27 @@ jobs:
       run: |
         cd dev
         chmod +x gradlew
+        echo "::group::Building Liberty.  This will take approx. 30 minutes"
         ./gradlew cnf:initialize assemble
+        echo "::endgroup::"
         cd .. && zip -rq openliberty-image.zip dev/build.image/wlp/
     - name: Upload liberty image
       uses: actions/upload-artifact@v2
       with:
         name: openliberty-image
         if-no-files-found: error
         path: openliberty-image.zip
-  unit_tests:
+  unit_tests: #Unit tests always run linux, but could depends on setup to determine jdk
     name: Unit Tests
-    runs-on: ubuntu-latest
+    needs: setup
+    runs-on: ubuntu-18.04
     timeout-minutes: 60
     steps:
-    - uses: actions/checkout@v2
-#    - uses: testspace-com/setup-testspace@v1
-#      with:
-#        domain: ${{ github.repository_owner }}  
+    - uses: actions/checkout@v2 
     - name: Set up Java
       uses: joschi/setup-jdk@v2
       with:
-        java-version: 11
+        java-version: ${{ needs.setup.outputs.test-java }}
```

Comment:
for now lets leave unit tests hardcoded to java 11. In our existing RTC setup we only ever run unit tests on java 11, so for now it should be sufficient to leave that behavior in place.